src/main/kotlin/com/jetbrains/snakecharm/string_language/SmkSLFileType.kt

```diff
@@ -0,0 +1,14 @@
+package com.jetbrains.snakecharm.string_language
+
+import com.intellij.openapi.fileTypes.LanguageFileType
+import javax.swing.Icon
+
+object SmkSLFileType : LanguageFileType(SmkStringLanguage) {
+    override fun getIcon(): Icon? = null
+
+    override fun getName() = "SmkSL"
+
+    override fun getDefaultExtension() = "smkStringLanguage"
+
+    override fun getDescription() = "Snakemake string language file"
```

Comment:
The description is shown in injection dialog and isn't very clear, lets use "Snakemake Formatted String"

![image](https://user-images.githubusercontent.com/72933/62865187-7288aa00-bd16-11e9-99f6-ad0ad5d5bfa9.png)